data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/IsmPolicyManagementStrategy.java

```diff
@@ -13,7 +13,7 @@
 
 interface IsmPolicyManagementStrategy {
 
-    Optional<String> checkAndCreatePolicy() throws IOException;
+    Optional<String> checkAndCreatePolicy(final String indexAlias) throws IOException;
```

Comment:
These changes do not have corresponding changes to the tests and are now failing.

Reply:
Fixed the tests for OpenSearchSinkIT